signer/local_cosigner.go

```diff
@@ -28,44 +28,41 @@ type LastSignStateStruct struct {
 	LastSignState *SignState
 }
 
-type LocalCosignerConfig struct {
-	// CosignerKey CosignerKey - removed
-	SignState *SignState
-	// RsaKey      rsa.PrivateKey - removed
-	Peers   []CosignerPeer
-	Address string
-	// localsigner is passed as parameter.
-	// Total       uint8 - removed
-	// Threshold   uint8 - removed
-}
-
 // LocalCosigner responds to sign requests using their share key
 // LocalCosigner "embeds" the Threshold signer.
 // LocalCosigner maintains a watermark to avoid double-signing via the embedded LastSignStateStruct.
 // LocalCosigner signing is thread safe by embedding *LastSignStateStruct which contains LastSignStateMutex sync.Mutex.
 type LocalCosigner struct {
 	LastSignStateStruct *LastSignStateStruct
+	id                  int
```

Comment:
This is a bit strange or? Might be me who are mixing ids up but why should LocalCosigner has an id?

Reply:
This was added to simplify `LocalCosigner` tests such as `TestLocalCosignerGetID` since they then do not require constructing a `ThresholdEd25519Signature`. But for single source of truth for the ID we can keep the `ThresholdEd25519Signature` as owner. Will update

Reply:
Changed back to how you had it. Constructed minimal threshold signer soft for the tests